examples/Tutorial.ipynb

```diff
@@ -0,0 +1,1521 @@
+      "Iris_CV_example_iris_MultinomialNB.results\n",
+      "Iris_CV_example_iris_MultinomialNB.results.json\n",
+      "Iris_CV_example_iris_MultinomialNB_fold1.model\n",
+      "Iris_CV_example_iris_MultinomialNB_fold10.model\n",
+      "Iris_CV_example_iris_MultinomialNB_fold2.model\n",
+      "Iris_CV_example_iris_MultinomialNB_fold3.model\n",
+      "Iris_CV_example_iris_MultinomialNB_fold4.model\n",
+      "Iris_CV_example_iris_MultinomialNB_fold5.model\n",
+      "Iris_CV_example_iris_MultinomialNB_fold6.model\n",
+      "Iris_CV_example_iris_MultinomialNB_fold7.model\n",
+      "Iris_CV_example_iris_MultinomialNB_fold8.model\n",
+      "Iris_CV_example_iris_MultinomialNB_fold9.model\n",
+      "Iris_CV_example_iris_MultinomialNB_predictions.tsv\n",
+      "Iris_CV_example_iris_RandomForestClassifier.log\n",
+      "Iris_CV_example_iris_RandomForestClassifier.results\n",
+      "Iris_CV_example_iris_RandomForestClassifier.results.json\n",
+      "Iris_CV_example_iris_RandomForestClassifier_fold1.model\n",
+      "Iris_CV_example_iris_RandomForestClassifier_fold10.model\n",
+      "Iris_CV_example_iris_RandomForestClassifier_fold2.model\n",
+      "Iris_CV_example_iris_RandomForestClassifier_fold3.model\n",
+      "Iris_CV_example_iris_RandomForestClassifier_fold4.model\n",
+      "Iris_CV_example_iris_RandomForestClassifier_fold5.model\n",
+      "Iris_CV_example_iris_RandomForestClassifier_fold6.model\n",
+      "Iris_CV_example_iris_RandomForestClassifier_fold7.model\n",
+      "Iris_CV_example_iris_RandomForestClassifier_fold8.model\n",
+      "Iris_CV_example_iris_RandomForestClassifier_fold9.model\n",
+      "Iris_CV_example_iris_RandomForestClassifier_predictions.tsv\n",
+      "Iris_CV_example_iris_SVC.log\n",
+      "Iris_CV_example_iris_SVC.results\n",
+      "Iris_CV_example_iris_SVC.results.json\n",
+      "Iris_CV_example_iris_SVC_fold1.model\n",
+      "Iris_CV_example_iris_SVC_fold10.model\n",
+      "Iris_CV_example_iris_SVC_fold2.model\n",
+      "Iris_CV_example_iris_SVC_fold3.model\n",
+      "Iris_CV_example_iris_SVC_fold4.model\n",
+      "Iris_CV_example_iris_SVC_fold5.model\n",
+      "Iris_CV_example_iris_SVC_fold6.model\n",
+      "Iris_CV_example_iris_SVC_fold7.model\n",
+      "Iris_CV_example_iris_SVC_fold8.model\n",
+      "Iris_CV_example_iris_SVC_fold9.model\n",
+      "Iris_CV_example_iris_SVC_predictions.tsv\n",
+      "Iris_CV_summary.tsv\n",
+      "\n",
+      "iris/test:\n",
+      "example_iris_features.jsonlines\n",
+      "\n",
+      "iris/train:\n",
+      "example_iris_features.csv       example_iris_features.jsonlines\n"
+     ]
+    }
+   ],
+   "source": [
+    "!ls iris/**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that the subdirectory `iris/output` has been created and that it contains several new files:\n",
+    "- a plain text `.results` file that contains human-readable output of the results for each learner in our list\n",
```

Comment:
Nitpick: There's no comma after the first bullet point (whereas there is one for every other bullet point after it that needs it). Also, I think typically there would be an "and" on the line preceding the last bullet point after the comma.

However, personally, I think all the commas should be dropped (and no "and" added as mentioned above).